app/views/layouts/application.html.erb

```diff
@@ -4,10 +4,54 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Crs2000</title>
+    <title>CRS2000</title>
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
     <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
     <%= csrf_meta_tags %>
+    <style type="text/css">
+      .navbar {
+        border-bottom: 5px solid black;
+        border-bottom-color: <%= primary_color %>;
+      }
+      .btnprimary {
```

Comment:
Don't you mean to say `.btn.btn-primary` or `.btn-primary`? i don't think bootstrap has a `.btnprimary` class.